core-bundle/src/Resources/contao/library/Contao/Environment.php

```diff
@@ -467,7 +467,7 @@ protected static function base()
 	 */
 	protected static function host()
 	{
-		return static::get('httpHost');
+		return preg_replace('/:\d+$/', '', static::get('httpHost'));
```

Comment:
This is a necessary BC break, otherwise there is no way to make an extension compatible with both Contao 4.9 (and below) and Contao 4.10 (and above). If someone explicitly needs the hostname with port number, they can use `Environment::get('httpHost')` or `{{env::http_host}}`, which also works in all Contao 4 versions.